package.json

```diff
@@ -16,7 +16,7 @@
   "packageManager": "pnpm@10.17.0+sha512.fce8a3dd29a4ed2ec566fb53efbb04d8c44a0f05bc6f24a73046910fb9c3ce7afa35a0980500668fa3573345bd644644fa98338fa168235c80f4aa17aa17fbef",
   "engines": {
     "//": "Update @types/node to match the highest node version here",
-    "node": ">=20 <=22",
+    "node": ">=22 <=24",
```

Comment:
Waarom toch nog 22? Misschien heb je de non-functional requirement net verkeerd gereverse-engineered: https://github.com/nl-design-system/documentatie/pull/3234/changes#r2627329192 (of misschien ben ik vergeten waarom toch een oude versie ondersteund moet blijven?)